Dockerfile

```diff
@@ -82,8 +82,8 @@ RUN mkdir -p ${APP_ROOT} && \
     curl -L https://github.com/ManageIQ/manageiq-ui-self_service/tarball/${REF} | tar vxz -C ${SSUI_ROOT} --strip 1
 
 ## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
-# ADD . ${APP_ROOT}
-RUN curl -L https://github.com/ManageIQ/manageiq/tarball/${REF} | tar vxz -C ${APP_ROOT} --strip 1
+ADD . ${APP_ROOT}
```

Comment:
Fair enough.

Then to your comment about approach, I guess I'm not in alignment for what you are suggesting.  Which, is to also say that I'm not a fan of the pulling down from Master to build the dockerfile.  Should any developers want to add something, then my thought is that they should clone the code locally apply whatever changes they wanted, then do a "docker build" and get their new image.